data-service/build.gradle

```diff
@@ -0,0 +1,3 @@
+description "Handler implementations for the Blox DataService"
```

Comment:
Yeah, handler is vague. I was thinking of the entrypoint interface that all Lambda functions implement, i.e. `RequestHandler`. Will change to just "Implementation of the Blox DataService"